src/snapred/backend/dao/request/CalibrationAssessmentRequest.py

```diff
@@ -19,3 +19,5 @@ class CalibrationAssessmentRequest(BaseModel):
     calibrantSamplePath: str
     useLiteMode: bool
     nBinsAcrossPeakWidth: int = Config["calibration.diffraction.nBinsAcrossPeakWidth"]
+    peakIntensityThreshold: float = Config["calibration.diffraction.peakIntensityThreshold"]
+    peakType: str = "Gaussian"
```

Comment:
should this be a literal?

Reply:
In pydantic a literal field is like an enum.  The user is able to modify this ti arbitrary float values.  The default value here is in case they don't.